src/xcode/ENA/ENA/Source/Services/Risk/Provider/__tests__/RiskProviderTests.swift

```diff
@@ -654,7 +654,7 @@ final class RiskProviderTests: CWATestCase {
 		
 		let didFailCalculateRiskExpectation = expectation(description: "expect didFailCalculateRisk not to be called")
 		
-		let expectedActivityStates: [RiskProviderActivityState] = [.riskRequested, .downloading, .detecting, .idle]
+		let expectedActivityStates: [RiskProviderActivityState] = [.riskManuallyRequested, .downloading, .detecting, .idle]
```

Comment:
I'm wondering if we could keep one test that is `userInitiated` and add one that is not so we can test that if it is not initiated by the user the activity state is set to `. riskRequested`?